packages/admin-ui/ui/src/hooks/use-edit-product-actions.tsx

```diff
@@ -74,8 +74,9 @@ const useEditProductActions = (productId: string) => {
     updateVariant.mutate(
       {
         variant_id: id,
-        ...removeNullish(payload),
+        ...removeFalsy(payload),
```

Comment:
question: @adrien2p @pKorsholm I am currently fiddling around with a reported bug around updating inventory quantities. Can you remember the reason for us using `removeFalsy` here? It will remove all number properties (e.g. `inventory_quantity`) that are attempted to be set to `0`. 

I have a PR ready, but I just want to make sure I don't introduce unexpected issues.

Reply:
In the current state, you can't update any number properties to 0, including all measurements + inventory.

Reply:
I dont recall but indeed it strips every falsy values, 0 being considered as false as well.

Reply:
RemoveNullish will strip all null or undefined which seems better no?

Reply:
I want to revert this to `removeNullish`.

Reply:
In that case null or undefined will not be assigned, is it what we want? Looks alright to me, but what are we really trying to achieve here? Undefined is stripped by the api i think and null could allow to reset a field that is nullable right?

Reply:
I can't immediately think of any properties that we would like to remove with `undefined` on variants. But if so, we would need a `removeNull`